README.md

```diff
@@ -13,6 +13,8 @@ Dependency Analytics is powered by [Snyk Intel Vulnerability DB](https://snyk.io
 - Suggests a project level license, check for conflicts between dependency licences
 - AI based guidance for additional, alternative dependencies
 
+ > :warning: **Disclaimer**: Dependency Analytics does not analyse **dev/test** dependencies.
```

Comment:
Do we need warning and disclaimer, just disclaimer should be good enough.

Reply:
okay, let's remove the warning then.